src/main/java/org/example/Seasons.java

```diff
@@ -0,0 +1,42 @@
+package org.example;
+
+import java.time.Month;
+
+public class Seasons {
+    public static class SeasonsOfDeclare {
+        enum Season {
```

Comment:
Dlaczego twój enum jest zagnieżdżony w dwie klasy? jest na to jakieś uzasadnienie? Nie lepiej zrobić public enum Seasons {tutaj miesiące} ? A metodę do wyliczania w innej klasie?